composer.json

```diff
@@ -6,7 +6,9 @@
         "php-http/httplug": "^1.0",
         "php-http/multipart-stream-builder": "^0.1",
         "php-http/message": "^1.0",
-        "php-http/discovery": "^1.0"
+        "php-http/client-common": "^1.0",
+        "php-http/discovery": "^1.0",
+        "webmozart/assert": "^1.2@dev"
```

Comment:
I was not feeling very comfortable with `@dev` version on a stable release. I suggest to upgrade to the stable release before to merge to `master` branch, but this is up to you.